src/zc/buildout/buildout.txt

```diff
@@ -1264,10 +1264,10 @@ Expressions must not contain either the ``#`` or the ``;`` character.
 Adding and removing options
 ---------------------------
 
-We can append and remove values to an option by using the + and -
+We can append and remove values to an option by using the ``+`` and ``-``
 operators.
 
-This is illustrated below; first we define a base configuration.
+This is illustrated below; first we define a base configuration::
```

Comment:
I agree.  While I generally prefer using ``::`` myself, I wanted to point out that the change wasn't strictly necessary since it's already recognized as a Doctest block:

http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#doctest-blocks

(Thinking about ReST while editing in MD makes me sad.  Silly GitHub.)